cvat-core/src/organization.ts

```diff
@@ -332,7 +332,7 @@ Object.defineProperties(Organization.prototype.leave, {
                 const result = await serverProxy.organizations.members(this.slug, 1, 10, {
                     filter: JSON.stringify({
                         and: [{
-                            '==': [{ var: 'user' }, user.id],
+                            '==': [{ var: 'user' }, user.username],
```

Comment:
Why do we need the change?

Reply:
I think  we need this change because of [this line](https://github.com/opencv/cvat/blob/develop/cvat/apps/organizations/views.py#L64). Searching by id isnt working, but searching by username works fine.